challenges.yml

```diff
@@ -1699,3 +1699,448 @@ extending_smart_watcher_2:
           },
           ```
       correct: a
+
+extending_css_transitions:
+  question: |
+    Imagine I have a simple component that can show a message when one of our
+    customers have bought too much pizza.
+
+    ```vue
+    <template>
+        <div :class="{
+                [$style.component]: true,
+                [$style.tall]: showDisclaimer,
+            }"
+        >
+            <button @click="addPizzaToCart" />
+
+            <p v-show="showDisclaimer">
+                <strong>Disclaimer</strong>: Eating too much pizza can be
+                bad for your help!
+            </p>
+        </div>
+    </template>
+
+    <script>
+    import { buyPizza } from '@/services/buy-pizza';
+
+    export default {
+        name: 'PizzaCheckout',
+        data() {
+            return {
+                showDisclaimer: false,
+            };
+        },
+        methods: {
+            async addPizzaToCart() {
+                await buyPizza();
+            }
```

Comment:
It's not super important, but we're missing some fake logic here to actually set `showDisclaimer` to true.

Maybe, instead of using `byPizza`, we have a `pizzas = []` data, and we have this:

```
addPizzaToCart() {
    this.pizzas.push({ ... });

    if (this.pizzas.length > 8) {
        this.showDisclaimer = true;
    }
}
```